README.md

```diff
@@ -31,7 +31,44 @@ brew install ethereum-optimism/tap/supersim
 
 Download the executable for your platform from the [GitHub releases page](https://github.com/ethereum-optimism/supersim/releases).
 
-### 3. Start `supersim` in vanilla mode
+### 3. Configure `supersim` (Optional)
+
```

Comment:
this readme section is great but can we move it down to a separate section `## Configuration` after `## First Steps`. The Getting Started should stay as short as possible